lizrd/support/sync_and_version.py

```diff
@@ -76,11 +76,21 @@ def rsync_to_remote(host, local_dir):
         raise Exception(f"[RSYNC ERROR]: An error occurred during rsync: {str(e)}")
 
 
+def connection_to_athena_nick(connection):
```

Comment:
Personally i follow convention similar to:
```suggestion
def get_user_folder_on_cluster(connection):
```